### PR TITLE
Create abstractHunter.json

### PR DESCRIPTION
**Pull request by abstractHunter**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

### CODE REVIEW

File: data/abstractHunter.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+      "name": "Let's connect on LinkedIn",
+      "url": "https://www.linkedin.com/in/yao-daniel-attitsogbe/",
+      "icon": "FaLinkedinIn"
+    },
+    {
+      "group": "Dev",
+      "name": "My Tublian profile",
+      "url": "https://www.tublian.com/profile/abstractHunter",
+      "icon": "FaGlobe"
+    },
```
Comment (Sindhuinti) [inline data/abstractHunter.json:28]:
```suggestion
    }
```

**Review by Sindhuinti — CHANGES_REQUESTED** [review]
Please remove this comma (not needed)

**Comment by abstractHunter** [comment]
Hello, it is done now.

**Review by Sindhuinti — APPROVED** [review]
Looks good 👍

**Review by sital002 — CHANGES_REQUESTED** [review]
It looks like you already have created a biodrop profile uaing forms here is the link for it https://www.biodrop.io/abstractHunter there is no point in making it again with json. You can alwats edit your profile using forms  https://www.biodrop.io/docs/how-to-guides/editing-forms 
I will be closing this Pull Request.

**Pull request closed without merging**